website/en/docs/types/utilities.md

```diff
@@ -145,6 +145,8 @@ type Obj = {
 type MappedObj = $ReadOnly<$ObjMap<Obj, TypeFn>> // Still read-only
 ```
 
+> Note: $ReadOnly is only for making read-only object types. See the Array docs for [how to type read-only arrays](../arrays/#toc-readonlyarray).
```

Comment:
You should mention the name `$ReadOnlyArray` here in case someone searches the Utility Types page for that name. I suggest wording the second sentence like this:

~~~markdown
See the Array docs for how to [type read-only arrays](../arrays/#toc-readonlyarray) with `$ReadOnlyArray`.
~~~